DigitalLearningSolutions.Data.Migrations/202201111458_LearningResourceReferencesOriginalRatingField.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+    [Migration(202201111458)]
+    public class LearningResourceReferencesOriginalRatingField : Migration
+    {
+        public override void Up()
+        {
+            Alter.Table("LearningResourceReferences")
+                .AddColumn("OriginalRating").AsDecimal(2, 1).NotNullable().WithDefaultValue(0);
+        }
+        public override void Down()
+        {
+            Delete.Column("OriginalRating").FromTable("OriginalRating");
```

Comment:
Ah, and the table's called LearningResourceReferences. Sorry I didn't spot that earlier.

Reply:
Oh dear. Rush job, sorry. Fixed, now.